.github/workflows/pullrequests.yml

```diff
@@ -0,0 +1,30 @@
+name: Pull Request Checks
+
+on:
+  pull_request:
```

Comment:
FYI that - if desired - you can limit this workflow to just running when files is specific directories are modified.

```
    paths:
      - '.github/workflows/pullrequests.yml'
      - 'pkg/**'
```